src/htmldocvisitor.cpp

```diff
@@ -353,23 +353,27 @@ void HtmlDocVisitor::visit(DocEmoji *s)
 
 void HtmlDocVisitor::writeObfuscatedMailAddress(const QCString &url)
 {
-  m_t << "<a href=\"#\" onclick=\"location.href='mai'+'lto:'";
-  if (!url.isEmpty())
-  {
-    const char *p = url.data();
-    uint size=3;
-    while (*p)
+  if (!Config_getBool(OBFUSCATE_EMAILS)) {
```

Comment:
Please follow doxygen's coding style, i.e. matching braces should be aligned on the same column. I had already fixed this, but your force commit undid my changes, so please do them yourself this time.

Reply:
Done